config/opal_mca.m4

```diff
@@ -145,15 +141,21 @@ AC_DEFUN([OPAL_MCA],[
         IFS="${IFS}$PATH_SEPARATOR,"
         msg=
         for item in $enable_mca_direct; do
-            type="`echo $item | cut -f1 -d-`"
-            comp="`echo $item | cut -f2- -d-`"
+            type="`echo $item | cut -s -f1 -d-`"
+            comp="`echo $item | cut -s -f2- -d-`"
             if test -z $type || test -z $comp ; then
-                AC_MSG_ERROR([*** The enable-mca-direct flag requires a
-*** list of type-component pairs.  Invalid input detected.])
-            else
-                AS_VAR_SET([AS_TR_SH([DIRECT_$type])], [AS_TR_SH([$comp])])
-                msg="$item $msg"
-            fi
+                AC_MSG_ERROR([enable-mca-direct requires a list of type-component pairs (ex. --enable-mca-direct=pml-ob1,smsc-xpmem)])
```

Comment:
Please fix the indenting here -- made it very confusing to read in the Github UI (perhaps these accidentally are tabs instead of spaces...?).

Reply:
Yeah. Need to fix my default emacs config to not use tabs ever. Freaking tabs are useless garbage :). Fixed.